socorro/lib/search_common.py

```diff
@@ -54,6 +54,14 @@
 }
 
 
+# Default date range for searches in days
+DEFAULT_DATE_RANGE = 7
+
+
+# Maximum date range for searches in days
+MAXIMUM_DATE_RANGE = 365
```

Comment:
I took these out of configuration and made them constants. We're never going to change these. Seems like they make for better constants.